qa-test3.planx-pla.net/portal/gitops.json

```diff
@@ -49,38 +49,35 @@
     "index": {
       "introduction": {
         "heading": "NIAID Data Ecosystem Datasets",
-        "text": "Disciplines within NIAID can have their own independent data models while maintaining interoperability of the data ecosystem. The data within projects are findable, accessible, interoperable, and reusable (FAIR), but it will be easier for each individual research community to work within the system.", 
+        "text": "Disciplines within NIAID can have their own independent data models while maintaining interoperability of the data ecosystem, facilitating individual research communities to work within the system. The data within all supported resources are findable, accessible, interoperable, and reusable (FAIR).", 
```

Comment:
this may be a me-problem but is "facilitating individual research communities to work within the system" correct english?

Reply:
I'm not married to the wording here.